etc/ci/performance/README.md

```diff
@@ -7,27 +7,21 @@ Servo Page Load Time Test
 
 # Basic Usage
 
-## Prepare the test runner
+If you want to run a performance test for your servo build. Simply go to the top-level servo directory and run `./mach test-perf`. The test result JOSN will be saved to `etc/ci/performance/output/`. You can compare two test results with the `test_differ.py` script. Run `python test_differ.py -h` for instructions.
 
-* Clone this repo
-* Download [tp5n.zip](http://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip), extract it to `page_load_test/tp5n`
-* Run `prepare_manifest.sh` to transform the tp5n manifest to our format
-* Install the Python3 `treeherder-client` package. For example, to install it in a virtualenv: `python3 -m virtualenv venv; source venv/bin/activate; pip install "treeherder-client>=3.0.0"`
+# Setup for CI machine
+
```

Comment:
This H1 is for the subsections "For Servo" and "For Gecko", I'll change them to "CI for Servo" to make it clear.